### DISCUSSION TITLE
While Compiling VRFCoordinatorMock "File Not Found" Error pops up

### DISCUSSION THREAD

**Discussion by akhilparakka** [smartcontractkit/full-blockchain-solidity-course-py#1634] [Q&A]
While Compiling VRFCoordinatorMock "File Not Found" Error pops up.
What I have tried:
- Tried Changing the version to v0.6
![Screenshot from 2022-08-10 14-26-03](https://user-images.githubusercontent.com/77268333/183860648-c95efb6a-b073-4f73-9e70-cad1a6c74573.png)

**Comment by cromewar** [comment]
Hello @Akhil-franko 
The problem occurs because you are not using the correct paths for the contracts on the version of Chainlink, most likely you are not using the correct version on `brownie-config.yaml` so:
1. Delete the build folder on your project.
2. Change the version of chainlink contracts on your config, to the latest one.
3. if nothing works, delete the `.brownie` folder on your home path and try again.
I hope this info might help, cheers.

**Reply by akhilparakka** [reply]
What I Tried
- Changed to the latest version
- deleted the build folder
Got A new error : TypeError: Contract "VRFCoordinatorV2Mock" should be marked as abstract.
![Screenshot from 2022-08-11 12-59-31](https://user-images.githubusercontent.com/77268333/184085911-375aaee2-88a0-4299-8345-c2bc4db3cf32.png)
![Screenshot from 2022-08-11 12-59-09](https://user-images.githubusercontent.com/77268333/184085936-0125735d-3e03-4c4d-81df-d040a1338ac7.png)

**Reply by cromewar** [reply]
The function must be overrided, otherwise the contract should be market as abstract, please take a look to the official Chainlink Contracts Github and read the contract you are importing from, you will notice `pendingRequestExists` has no implementation so it's your duty to implement it.
